docs/cn/MATH.md

```diff
@@ -0,0 +1,220 @@
+
+首先，安装这两种渲染器的**其中一个**：
+
+```sh
+npm un hexo-renderer-marked --save
+npm i hexo-renderer-pandoc --save # 或者 hexo-renderer-kramed
+```
+
+
+然后在 `next/_config.yml` 中将 `math` 的 `enable` 打开，并选择 `mathjax` 作为渲染引擎。
+
+```yml
+math:
+  enable: true
+  ...
+  engine: mathjax
+  #engine: katex
+```
+
+执行 Hexo 生成，部署，或者启动服务器：
+
+```sh
+hexo clean && hexo g -d
+# 或者 hexo clean && hexo s
+```
+
+### Katex
+
+Katex 渲染引擎相对于 MathJax 来说**大大提高了速度**，但是 Katex 所支持的东西没有 MathJax 全面，你可以从下面的相关链接中获取更多的信息。
+
+如果你选择使用 Katex 进行数学公式渲染，你需要使用 [hexo-renderer-markdown-it-plus](https://github.com/CHENXCHEN/hexo-renderer-markdown-it-plus) 或者 [hexo-renderer-markdown-it](https://github.com/hexojs/hexo-renderer-markdown-it) 这两种渲染器的其中一个。
+
+首先，安装这两种渲染器的**其中一个**：
+
+```sh
+npm un hexo-renderer-marked --save
+npm i hexo-renderer-markdown-it-plus --save
+# 或者 hexo-renderer-markdown-it
+```
+
+
+然后在 `next/_config.yml` 中将 `math` 的 `enable` 打开，并选择 `mathjax` 作为渲染引擎。
+
+```yml
+math:
+  enable: true
+  ...
+  engine: mathjax
+  #engine: katex
+```
+
+执行 Hexo 生成，部署，或者启动服务器：
+
+```sh
+hexo clean && hexo g -d
+# 或者 hexo clean && hexo s
+```
+
+#### 如果你使用 hexo-renderer-markdown-it
+
+如果你使用 `hexo-renderer-markdown-it`，你还需要为其加上 `markdown-it-katex` 作为插件：
+
+```
+npm i markdown-it-katex --save
+```
+
+然后在 `hexo/_config.yml` 中将 `markdown-it-katex` 作为插件写入 `hexo-renderer-markdown-it` 的配置中：
+
+```yml
+markdown:
+  render:
+    html: true
+    xhtmlOut: false
+    breaks: true
+    linkify: true
+    typographer: true
+    quotes: '“”‘’'
+  plugins:
+    - markdown-it-katex
+```
+
+#### 已知的问题
+
+1. 首先请查阅 Katex 的 [Common Issue](https://github.com/Khan/KaTeX#common-issues)
+2. 块级公式(例如 `$$...$$`)必须位于空行。\
+   即在开头的 `$$` 前和在结尾的 `$$` 后不能有除了空白字符以外的其他字符。([#32comment](https://github.com/theme-next/hexo-theme-next/pull/32#issuecomment-357489509))
+3. 不支持 Unicode。([#32comment](https://github.com/theme-next/hexo-theme-next/pull/32#issuecomment-357489509))
+4. 行内公式(例如 `$...$`)在开头的 `$` 后面和结尾的 `$` 前面**不能含有空格**。([#32comment](https://github.com/theme-next/hexo-theme-next/pull/32#issuecomment-357489509))
+5. 如果你在文章的各级标题中(例如 `## 标题`)使用公式。\
+   那么文章目录中的这个标题会出现 3 次未渲染的公式代码([#32comment](https://github.com/theme-next/hexo-theme-next/pull/32#issuecomment-359018694))
+6. 如果你在文章 Title 中使用公式，那么公式将不会被渲染。([#32comment](https://github.com/theme-next/hexo-theme-next/pull/32#issuecomment-359142879))
+
+
+我们目前使用的 Katex 版本为 0.7.1，这里面可能有某些问题是因为 Katex 版本老旧导致的；
+
+但是，就像上面所说的，数学公式的渲染必须依靠渲染器来支持，目前的 Katex 相关的渲染器仅支持到 Katex 0.7.1；
+
+我们会持续关注相关渲染器的更新，如果有渲染器支持更高版本的 Katex，我们会及时更新我们的 Katex 版本。
+
+### 相关链接
+
+* [Katex 与 MathJax 渲染速度对比](https://www.intmath.com/cg5/katex-mathjax-comparison.php)
+* [Katex 支持的功能列表](https://khan.github.io/KaTeX/function-support.html)
+
+<h2 align="center">相关配置说明</h2>
+
+注意，在修改配置选项时，**不要更改配置的缩进**；
+
+目前，NexT 的所有配置都采用**2 空格的缩进**；
+
+如果配置的内容接在冒号后面，那么内容和冒号之间必须有一个空格(例如`enable: true`)
+
+```yml
+# Math Equations Render Support
+math:
+  enable: false
+
+  # Default(flase) will load mathjax/katex srcipt EVERY PAGE
```

Comment:
> ...(false)

Reply:
That's the typo of `_config.yml`.

Will fix it right away.